addons/survey/tests/test_survey.py

```diff
@@ -39,19 +39,19 @@ def setUp(self):
         self.UserInput = self.env['survey.user_input']
         self.UserInputLine = self.env['survey.user_input_line']
 
-        self.survey1 = self.Survey.sudo(self.survey_manager).create({'title': "S0", 'page_ids': [(0, 0, {'title': "P0"})]})
+        self.survey1 = self.Survey.sudo(self.survey_manager.id).create({'title': "S0", 'page_ids': [(0, 0, {'title': "P0"})]})
         self.page1 = self.survey1.page_ids[0]
+        self.question = self.Question.sudo(self.survey_manager.id).create({'page_id': self.page1.id, 'question': 'Q0'})
```

Comment:
If question is used only in test_00_create_minimal_survey then why not kept there(Instead of storing at class scope)